binaries/streamer_binaries/__init__.py

```diff
@@ -31,3 +38,11 @@
 packager = os.path.join(_dir_path, 'packager-{}-{}'.format(_os, _cpu))
 """The path to the installed Shaka Packager binary."""
 
+# Special overrides for Ubuntu builds with hardware encoding support.
+# These are not static binaries, and so they must be matched to the distro.
+if _os == 'linux':
+  if distro.id() == 'ubuntu':
+    if distro.version() in _ubuntu_versions_with_hw_encoders:
+      suffix = '-ubuntu-{}'.format(distro.version())
```

Comment:
nit: we should be able to add them like this:
```python
'-ubuntu-' + distro.version()
```

Reply:
Done